implementations/elixir/ockam/ockam_hub/config/config.exs

```diff
@@ -2,4 +2,11 @@ import Config
 
 config :logger, level: :info
 
+config :telemetry_influxdb,
+  host: System.get_env("INFLUXDB_HOST"),
+  port: System.get_env("INFLUXDB_PORT"),
+  bucket: System.get_env("INFLUXDB_BUCKET"),
+  org: System.get_env("INFLUXDB_ORG"),
+  token: System.get_env("INFLUXDB_TOKEN")
+
```

Comment:
The way the Azure image is setup this token is stored in a file at this path `/mnt/secrets/influx/token`
Our elixir should must read it from that path.

Secrets, that are stored in environment variables, often accidentally to end up in system logs.